CarService/CarService.Users.Api/Models/Domain/User.cs

```diff
@@ -5,6 +5,9 @@ namespace CarService.Users.Api.Models.Domain
     public class User
     {
         public Guid Id { get; set; }
+        public string Username { get; set; }
+        public string Password { get; set; }
```

Comment:
Password shouldn't be stored in DB. Please use password hash, and compare with it